.evergreen/.evg.yml

```diff
@@ -740,6 +928,13 @@ buildvariants:
   tasks:
     - name: "plain-auth-test"
 
+- matrix_name: "aws-auth-test"
+  matrix_spec: { jdk: "jdk11" }
+  display_name: "MONGODB-AWS Auth test"
+  run_on: ubuntu1804-test
```

Comment:
Do these need to be run on `ubuntu1804` or will any (aws hosted) distro work?

Reply:
It would be preferable to run on the same host types as the other tests.

Reply:
I'm thinking about Python's testing where we would want to use a distro that has the python toolchain on it: `rhel62-small`, `ubuntu1604-test`, or `debian92-test`. We don't use `ubuntu1804-test` currently.

Reply:
The ECS container that you'll need to SSH into is 18.04 so I don't think you'd be able to copy over binaries from other distros to the ECS container. Because of this, the scripts were only tested on 18.04. That being said you can still test it on whichever distro you want but you'd just have to keep those points in mind.

Reply:
I'll add a note to the testing SPEC for this too.

Reply:
Hmm. I don't see the `3. ECS instance` test in this PR.  Is that test still a todo?

Reply:
This is uncommented now.